tests/test_connect_negative.c

```diff
@@ -121,9 +121,9 @@ int main(void) {
       cmocka_unit_test(test_connect_account_missing),
       cmocka_unit_test(test_connect_user_missing),
       cmocka_unit_test(test_connect_password_missing),
-      cmocka_unit_test(test_connect_invalid_database),
-      cmocka_unit_test(test_connect_invalid_schema),
-      cmocka_unit_test(test_connect_invalid_warehouse),
+      //cmocka_unit_test(test_connect_invalid_database),
```

Comment:
Can you create a JIRA to re-enable these tests and assign it to me for the next sprint?

Reply:
@rvishureddy created it and I CCed you on SNOW-85815